helm/README.md

```diff
@@ -0,0 +1,107 @@
+# Agent Check: helm
+
+## Overview
+
+This check monitors [helm][1] through the Datadog Agent.
```

Comment:
We believe this check should be run as a [cluster check](https://docs.datadoghq.com/agent/cluster_agent/clusterchecks/), we only need one instance of the check running in the whole cluster, let's make this clear in the documentation